app/controllers/api/groups_controller.rb

```diff
@@ -56,4 +56,8 @@ def accessible_records
     Queries::ExploreGroups.new
   end
 
+  def resource_options(scope:, serializer:, root:)
```

Comment:
Seems likely there are times where there will be an event to return from the groups controlller, right? 

Does this code mean we won't serialize out a event for groups controller?